corehq/apps/cloudcare/templates/cloudcare/partials/case_list/bootstrap5/tile_grouped_item.html

```diff
@@ -37,12 +37,12 @@
             <div class="<%- prefix %>-grid-style-<%- index %> box">
               <% if (styles[index].displayFormat === 'Image') {
                 if(resolveUri(datum)) { %>
-                  <img alt="<%- altText[index] %>" class="module-icon" style="max-width:100%; max-height:100%;" src="<%- resolveUri(datum) %>"/>  {# todo B5: inline style #}
+                  <img alt="<%- altText[index] %>" class="module-icon" src="<%- resolveUri(datum) %>"/>
                 <% } %>
               <% } else if (styles[index].displayFormat === 'ClickableIcon') {
                 if(resolveUri(datum)) { %>
                   <button class="clickable-icon">
-                    <img alt="<%- altText[index] %>" class="module-icon" src="<%- resolveUri(datum) %>" style="max-width:100%; max-height:100%;"/>  {# todo B5: inline style #}
+                    <img alt="<%- altText[index] %>" class="module-icon" src="<%- resolveUri(datum) %>" />
                     <i class="fa fa-spin fa-spinner" style="display:none"></i>  {# todo B5: inline style #}
```

Comment:
is this `todo` comment still needed?

Reply:
yeah, I'm going to fix that one in a different PR